### PR TITLE
Feature(CB2-11329): Increase start values on test number seed data

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Increase start values on test number seed data

Increase start values on test number seed data for the following:
- system number
- znumber
- tnumber
- cnumber (trailer Id)
- plate serial number
[CB2-11329](https://dvsa.atlassian.net/browse/CB2-11329)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by LGin-BJSS — DISMISSED** [review]

File: src/config/config.yml [hunk 1]
```diff
@@ -63,19 +63,19 @@ testNumberinitialValue:
...
+  plateSerialNumber: '4000000'
 ZNumberInitialValue:
-  zNumber: 1000000Z
+  zNumber: 4000000Z
   zNumberLetter: Z
-  sequenceNumber: 1000000
+  sequenceNumber: 4000000
 TNumberInitialValue:
-  tNumber: 020000T
+  tNumber: 4000000T
```
Comment (cb-cs) [inline src/config/config.yml:78]: Do you know if there is a reason the T number is pre-pended by a 0?

File: src/config/config.yml [hunk 2]
```diff
@@ -63,19 +63,19 @@ testNumberinitialValue:
...
+  trailerId: C4000000
   trailerLetter: C
-  sequenceNumber: 530000
+  sequenceNumber: 4000000
 systemNumberInitialValue:
-  systemNumber: '10000000'
+  systemNumber: '4000000'
 plateSerialNumberInitialValue:
-  plateSerialNumber: '0'
+  plateSerialNumber: '4000000'
```
Comment (naathanbrown) [inline src/config/config.yml:72]: We need to check product are okay with this being jumped. Apparently they really care about this value for proof the plate is real etc. Sorry I only found out last week

Comment (LGin-BJSS) [reply src/config/config.yml:72]: have we seen any conflicts with numbers on plate serial numbers? can this just stay at 0?

Comment (Daniel-Searle) [reply src/config/config.yml:72]: Checked with Mel, reckons it's best we leave the plates one alone! 😄

**Review by kcarr-bjss — APPROVED** [review]

**Review by LGin-BJSS — APPROVED** [review]

**Review by RobWhitehouseBJSS — APPROVED** [review]

**Pull request merged**